src/test/java/org/json/simple/JSONArrayTest.java

```diff
@@ -265,7 +265,7 @@ public void testObjectArrayToString() throws IOException {
 		assertEquals("null", JSONArray.toJSONString((Object[])null));
 		assertEquals("[]", JSONArray.toJSONString(new Object[0]));
 		assertEquals("[\"Hello\"]", JSONArray.toJSONString(new Object[] { "Hello" }));
-		assertEquals("[\"Hello\",12,[1,2,3]]", JSONArray.toJSONString(new Object[] { "Hello", Integer.valueOf(12), new int[] { 1, 2, 3 } }));
+		assertEquals("[\"Hello\",12,[1,2,3]]", JSONArray.toJSONString(new Object[] { "hello", Integer.valueOf(12), new int[] { 1, 2, 3 } }));
```

Comment:
This one should be "Hello" instead of "hello"

Reply:
Ok, I'll fix it! Thank You for the warning! :D